onnxruntime/core/graph/graph.cc

```diff
@@ -2987,6 +2991,15 @@ Status Graph::InlineFunction(Node& node) {
   return Status::OK();
 }
 
+void Graph::SyncWithFP16Initializer(const std::unordered_map<std::string, NodeArg*>& updated_input_map) {
```

Comment:
How does it relate to fp16?

Reply:
This method is not used any more. deleted.

---
In reply to: [434120880](https://github.com/microsoft/onnxruntime/pull/4113#discussion_r434120880) [](ancestors = 434120880)